src/routes/login/+page.server.ts

```diff
@@ -16,17 +17,10 @@ export const actions: Actions = {
 		const existingUser = await getUserWithoutSerialization(login);
 
 		if (!existingUser) {
-			const randomMs = Math.floor(Math.random() * 1000);
```

Comment:
Not sure what was the point of this code.

1. Why random wait?
2. The function is already async it will always return a promise, returning `new Promise(....)` just adds a bit more work for compiler.

Reply:
It was here to simulate request time, as immediate response means that the user doesn't exist.

When I think about it know, maybe it was too much precaution on my side :)